### PR TITLE
fix: better error messages and failed execution info

### PR DESCRIPTION
**Pull request by exu**
## Pull request description 

Fixing more edge cases in jobs execution

## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

**Review by vsukhin — APPROVED** [review]
lgtm

File: cmd/kubectl-testkube/commands/scripts/execution_renderer.go [hunk 1]
```diff
@@ -77,14 +78,14 @@ func (r ExecutionRawRenderer) Watch(execution testkube.Execution, writer io.Writ
...
+// - move all logic from execution, start, watch here to show final execution
 func (r ExecutionRawRenderer) renderDetails(execution testkube.Execution, writer io.Writer) error {
-	_, err := fmt.Fprintf(writer, "Name: %s, Status: %s, Duration: %s\n",
-		execution.Name,
-		*execution.ExecutionResult.Status,
-		execution.CalculateDuration(),
-	)
-
-	return err
+	ui.Writer = writer
```
Comment (vsukhin) [inline cmd/kubectl-testkube/commands/scripts/execution_renderer.go:85]: Is this thread safe? Write is a global var

**Review by vsukhin — COMMENTED** [review]
found one suspicious statement

File: cmd/kubectl-testkube/commands/scripts/execution_renderer.go (hunk 1 above)
Comment (exu) [reply cmd/kubectl-testkube/commands/scripts/execution_renderer.go:85]: @vsukhin FYI extracted this as separate issue https://github.com/kubeshop/testkube/issues/878

**Pull request merged**